packages/xo-web/src/xo-app/home/index.js

```diff
@@ -1147,11 +1185,16 @@ export default class Home extends Component {
               </div>
             )}
           </Col>
+
           <Col smallSize={6} mediumSize={2} className='text-xs-right'>
-            <Button onClick={this._expandAll}>
+            <Button onClick={this._expandAll} disabled={disableAction}>
```

Comment:
Do we want to use sorted tables in the home view?

Reply:
Yes, why not?

Reply:
Because i saw a duplication of home actions, the disable of some incompatible actions and changes that can be avoided by setting a special filter on the home view.